FEXCore/Source/Interface/Config/Config.json.in

```diff
@@ -338,6 +338,13 @@
           "Enables FEX's low-overhead sampling profile statistics.",
           "Requires a supported version of Mangohud to see the results"
         ]
+      },
+      "TraceProfiler": {
+        "Type": "bool",
+        "Default": "false",
+        "Desc": [
+          "Enables FEX's gpuvis trace profiler."
+        ]
```

Comment:
Since this is gpuvis-specific now, could you reflect this in the option name to avoid confusion? 

Also the description is hardly distinguishable from the build-time option; changing the wording from "enable profiler" to "enables profiling" should be clearer.

```suggestion
      },
      "EnableGpuvisProfiling": {
        "Type": "bool",
        "Default": "false",
        "Desc": [
          "Enables profiling when FEX was built with the gpuvis profiler backend."
        ]
```

Reply:
Updated with your words.